tools/ctl/ctl.go

```diff
@@ -216,6 +217,94 @@ var getResults = &cobra.Command{
 	},
 }
 
+var missingAttestations = &cobra.Command{
```

Comment:
As discussed, I'm dubious on this having much long-term value but I'm fine using it as a util for now. We should look to doing this diff on firestore results for smoketest vs actual rebuilds. This procedure could be used to assert consistency between the two data sources but I suspect we'd want that running as a cron and not in an adhoc CLI.

To reflect this, do you want to add a TODO to implement this diff in a command like get-result instead? diff-result?